andela/ah-bird-box-frontend
# #163388216 Users should be able to rate articles

#### What does this PR do?

This PR adds rating functionality for articles by authenticated users.

#### Description of Task to be completed?

Add tests
Add a rating functionality for the article

#### How should this be manually tested?

1. Checkout to the specific branch with `git checkout ft-articles-rating-163388216`
2. Run `npm install` to install all the dependencies of the application
3. Run `npm start`
4. Navigate to a single article page.
5. Check at the end of the article to find the rating.

#### Any background context you want to provide?

None

#### What are the relevant pivotal tracker stories?

[#163388216](https://www.pivotaltracker.com/story/show/163388216)

#### Screenshots (if appropriate):

![image](https://user-images.githubusercontent.com/5156758/55074584-b8174400-50a1-11e9-80f9-ba7112abe855.png)

![image](https://user-images.githubusercontent.com/5156758/55074618-cebd9b00-50a1-11e9-93c3-5c9ac31831b3.png)

#### Questions:

None

#### Reviewers:

@TooColline
@calebrotich10
@unah254
@bryan-munene
@hoslack
